src/index.css

```
@@ -1,63 +1,45 @@
+@import "tailwindcss/theme" prefix(wcpos);
+@import "tailwindcss/utilities" prefix(wcpos);
 
-#woocommerce-pos-settings table {
-	/* 1 */
-	text-indent: 0;
-	/* 2 */
-	border-color: inherit;
-	/* 3 */
-	border-collapse: collapse;
+/* Custom WordPress admin theme colors */
+@theme inline {
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)